contrib/gorilla/mux/mux_test.go

```diff
@@ -199,6 +199,43 @@ func TestAnalyticsSettings(t *testing.T) {
 	})
 }
 
+func TestSkippingOption(t *testing.T) {
+	skippingFunc := func(req *http.Request) bool {
```

Comment:
Can we just inline this into the option of `NewRouter`?

Reply:
Sure